contracts/utils/ExternalMulticall.sol

```diff
@@ -0,0 +1,57 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "./interfaces/IExternalMulticall.sol";
+
+/// @notice Contract that enables calls to external contracts to be batched
+/// @dev Refer to MakerDAO's Multicall.sol for a similar implementation
+contract ExternalMulticall is IExternalMulticall {
```

Comment:
This is essentially the base of a more flexible OwnableCallForwarder to be used by searchers